src/main/java/org/example/LessonDao.java

```
@@ -60,29 +55,37 @@ public List<Lesson> getAllLessons() throws SQLException {
              ResultSet resultSet = statement.executeQuery(query)) {
 
             while (resultSet.next()) {
-                int lessonId = resultSet.getInt(1);
-                String lessonName = resultSet.getString(2);
-                int homeworkId = resultSet.getInt(3);
-                String homeworkName = resultSet.getString(4);
-                String homeworkDescription = resultSet.getString(5);
+                int lessonIdIndex = 1;
+                int lessonNameIndex = 2;
+                int homeworkIdIndex = 3;
+                int homeworkNameIndex = 4;
+                int homeworkDescriptionIndex = 5;
+
+                int lessonId = resultSet.getInt(lessonIdIndex);
+                String lessonName = resultSet.getString(lessonNameIndex);
+                int homeworkId = resultSet.getInt(homeworkIdIndex);
+                String homeworkName = resultSet.getString(homeworkNameIndex);
+                String homeworkDescription = resultSet.getString(homeworkDescriptionIndex);
 
                 Homework homework = new Homework(homeworkId, homeworkName, homeworkDescription);
                 Lesson lesson = new Lesson(lessonId, lessonName, homework);
 
                 lessons.add(lesson);
             }
         } catch (SQLException e) {
-            throw new SQLException("Error retrieving lessons: " + e.getMessage());
+            System.err.println("Error retrieving lessons: " + e.getMessage());
```

throw an exception instead of using a print statement